receiver/kafkametricsreceiver/README.md

```diff
@@ -35,6 +35,7 @@ Metrics collected by the associated scraper are listed [here](metadata.yaml)
 
 Optional Settings (with defaults):
 
+- `cluster_alias` (default = kafka-cluster): Alias name of the cluster.
```

Comment:
To make sure I understand: The goal of this configuration option is to be able to set an attribute value for all relevant metrics that are emitted by this receiver, is that correct? 

If so, is there a reason we're not relying on a processor to add the attribute? Also, should this actually a resource attribute, rather then datapoint attribute?

Reply:
Correct, It's intended to provide a new attribute (`cluster_alias`) for all metrics. It will store a cluster identifier.
It's possible to achieve this using processors, But it's hard to manage when we have many clusters. for current implementation we need to maintain separate pipelines for each clusters. 

for example,
```yaml
...

processors:
  resource/kafka-1-label:
    attributes:
      - key: cluster_alias
        action: insert
        value: kafka-1
  resource/kafka-2-label:
    attributes:
      - key: cluster_alias
        action: insert
        value: kafka-2

service:
  pipelines:
    metrics/kafka-1:
      receivers:
        - kafkametrics/kafka-1
      processors: [resource/kafka-1-label]
      exporters: [signalfx]
    metrics/kafka-2:
      receivers:
        - kafkametrics/kafka-2
      processors: [resource/kafka-2-label]
      exporters: [signalfx]
```

it would be great if there is an optional cluster identifier that makes filtering metrics easier.

Reply:
And yes, I have added resource attribute for `cluster_alias`.

Reply:
Okay, I could see the potential value there. I'll defer to code owners for a more definitive response, but it's not a blocker for me.